osf/migrations/0137_add_fm_record_to_osfstorage_files.py

```diff
@@ -1,32 +1,58 @@
 # -*- coding: utf-8 -*-
 # Generated by Django 1.11.13 on 2018-09-26 15:18
 from __future__ import unicode_literals
+from math import ceil
+import logging
 
 from django.db import migrations, connection
 
+logger = logging.getLogger(__name__)
+
+increment = 100000
```

Comment:
I have 558372 OsfStorageFiles locally - INSERTS were divided into 16 batches though, probably because of the spread of the ids.

Reply:
might have to play with increment

Reply:
500k seemed fine.